docs/configuring-socorro.rst

```diff
@@ -8,6 +8,19 @@ Socorro produces graphs and reports, most are updated once per day.
 You must enter information about your releases into Socorro in order
 for this to work, and this information must match the incoming crashes.
 
+Configure Elasticsearch
+-----------------------
+
+Many features of Socorro depend on Elasticsearch, and to work you will first
```

Comment:
This is true but vague - what data, and why?  I would recommend adding some detail as to what this script actually does.